cargo-cyclonedx/src/config.rs

```diff
@@ -253,30 +234,55 @@ pub enum ParseMode {
     Lax,
 }
 
+/// What does the SBOM describe?
+#[derive(Copy, Clone, Debug, Default, PartialEq, Eq)]
+pub enum Describe {
```

Comment:
One possible expansion here that may help users understand these options better is to add the  `clap::ValueEnum` derive.

```rust
#[derive(clap::ValueEnum, Copy, Clone, Debug, Default, PartialEq, Eq, Serialize)]
#[serde(rename_all = "kebab-case")]
pub enum Describe {
    // ..
}
```

then the supported strings with their descriptions are displayed as well for the `--describe` option

Feel free to ignore if you think the help is already sufficient.

Reply:
Excellent idea! Done.